### PR TITLE
Feat: Onboard EPA Historical Air Quality dataset

### PR DESCRIPTION
**Pull request by nlarge-google**
## Description

* Dataset now utilizes individual clusters in order to resolve resource failures when running on the main cluster

## Checklist

Note: If an item applies to you, all of its sub-items must be fulfilled

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved
- [ ] I'm adding or editing a feature
  - [ ] I have added tests for the feature
- [x] I'm adding or editing a dataset
  - [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
  - [x] I put all my code inside  `datasets/epa_historical_air_quality` and nothing outside of that directory
- [x] I'm submitting a bugfix
- [x] I'm refactoring or cleaning up some code

### CODE REVIEW

**Comment by nlarge-google** [comment]
@adlersantos Please review these changes

**Comment by nlarge-google** [comment]
@adlersantos @happyhuman Please review the code changes.  The system now implements the new folder structure along with utilizing a cluster create and other changes to improve code.

**Review by adlersantos — CHANGES_REQUESTED** [review]
Let's merge these multiple DAGs into one that creates and deletes a single cluster

File: datasets/epa_historical_air_quality/pipelines/hap_daily_summary/pipeline.yaml [hunk 1]
```diff
@@ -1,207 +0,0 @@
...
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
----
-resources:
-
-  - type: bigquery_table
-    table_id: "hap_daily_summary"
-    description: "epaspc"
```
[the remark this answers is not in the document]
Comment (adlersantos) [reply datasets/epa_historical_air_quality/pipelines/hap_daily_summary/pipeline.yaml:20]: After generating the TF files, you should have very few files under the `infra` folder.

[the remark this answers is not in the document]
Comment (nlarge-google) [reply datasets/epa_historical_air_quality/pipelines/hap_daily_summary/pipeline.yaml:20]: Done

**Comment by nlarge-google** [comment]
@adlersantos @happyhuman Please review following latest changes.  Thanks!

**Review by adlersantos — APPROVED** [review]

**Pull request merged**